src/Microsoft.Health.Fhir.Core/Configs/ExportJobConfiguration.cs

```diff
@@ -42,6 +42,16 @@ public class ExportJobConfiguration
         /// </summary>
         public uint MaximumNumberOfResourcesPerQuery { get; set; } = 10000;
 
+        /// <summary>
+        /// For SQL export, controlls the number of parallel id ranges to gather to be used for parallel export.
+        /// </summary>
+        public int NumberOfParallelRecordRanges { get; set; } = 100;
+
+        /// <summary>
+        /// For SQL export, controlls the DOP (degree of parallelization) used by the coordinator to build sub-jobs.
+        /// </summary>
+        public int CoordinatorMaxDegreeOfParallelization { get; set; } = 4;
```

Comment:
I don't see the reason to provide these constants as long as there is no indication that we need to change them.

Reply:
`CoordinatorMaxDegreeOfParallelization` is the outlier here. I'm happy to leave that as is in code. It was the only static value left after the others were passed by config, so I was sticking to that pattern.

The other values like `NumberOfParallelRecordRanges` and `NumberOfParallelRecordRanges` are in config so our [E2E tests](https://github.com/microsoft/fhir-server/blob/feature/export/include-history-soft-delete/test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Export/StartupForExportTestProvider.cs#L46) can reduce these values for testing multiple job exports with